src/common.ts

```
@@ -0,0 +1,261 @@
+      other.#offset,
+      new Location(this.#line, this.#column),
+      new Location(other.#line, other.#column),
+    );
+  }
+}
+
+/**
+ * Used to track offsets and locations.
+ */
+export class LocationTracker {
+  #offset: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)